Frontend/src/Review/CreateReviewModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import "./CreateReviewModal.css";
+
+const CreateReviewModal = ({ isOpen, onClose, onSubmit }) => {
```

Comment:
hm is this the same code as in the 3rd commit in this PR?

Reply:
Yeah, I initially did not know why it was showing as uncommitted on my vs code. Now I realized the folder was created twice.